pre-lib-harddrive.sh

```diff
@@ -59,3 +55,22 @@ function prepare_tree() {
     umount "${MOUNT_DIR}"
     rm -rf "${MOUNT_DIR}"
 }
+
+# Format a complete disk for hdd payload source
+function format_whole_disk () {
+    local disk="$1"
+
+#    wipefs -a "${disk}"
+    sgdisk --zap-all ${disk}
+    mkfs.ext4 -F "${disk}"
+}
+
+
+# Format single partition on a disk for hdd payload source
+function format_single_partition () {
+    local disk="$1"
+
+    sgdisk --zap-all ${disk}
+    sgdisk --new=0:0:+12GiB ${disk}
```

Comment:
Shouldn't the partition size be also increased to 22GiB?

Reply:
Good point, let me check, we should probably parametrize the function.

Reply:
Oh, I guess we need the unallocated space for autopart.